napari/utils/_appdirs.py

```diff
@@ -4,14 +4,19 @@
 from typing import Callable, Optional
 
 import appdirs
+from packaging import version
 
-from napari._version import __version_tuple__
+from napari._version import __version__
+
+napari_version = version.parse(__version__)
+
+version_string = napari_version.base_version
+if napari_version.is_devrelease:
+    version_string += "dev"
 
 _appname = 'napari'
```

Comment:
Previously the version of this being used to set the config path was in _base as `Napari`, so this could be an issue with loading previous settings?

Reply:
(due to case sensitivity, old settings in `Napari` may not be found.)

Reply:
:facepalm: 
Appauthor is not a problem as it is applied only on Windows. But also APPNAME had `Napari`, not `napari`. 
:facepalm:

Reply:
Right, but won't anyone that has napari now have `Napari/settings.yaml` and when they install this they will get a new `napari/<version>/settings.yaml` along side?

Reply:
yes

Reply:
OK, i'm not getting a `/napari` so I think at least on macOS this isn't an issue. Also, it's a new thing with the hashes anyways, so no need to worry about backwards stuff.